component/service/src/main/java/org/exoplatform/social/rest/api/EntityBuilder.java

```diff
@@ -192,9 +192,14 @@ public static ProfileEntity buildEntityProfile(Profile profile, String restPath,
       User user = organizationService.getUserHandler().findUserByName(userEntity.getUsername());
       if (user != null) {
         userEntity.setIsInternal(user.isInternalStore());
+        if (user.getLastLoginTime().equals(user.getCreatedDate())) {
+          userEntity.setLastConnexion(null);
+        } else {
+          userEntity.setLastConnexion(String.valueOf(user.getLastLoginTime().getTime()));
+        }
```

Comment:
```suggestion
        if (!user.getLastLoginTime().equals(user.getCreatedDate())) {
                userEntity.setLastConnexion(String.valueOf(user.getLastLoginTime().getTime()));
        }
```
Is it sufficient ?

Reply:
For this PR , we will don't treat the enrollment date , we need just to compare the last connection date with the creation date to know if the user has logged or not
so yes it sufficient.

Reply:
yes, updated!